src/adapter/src/coord/dataflows.rs

```diff
@@ -102,9 +103,15 @@ pub enum ExprPrepStyle<'a> {
     OneShot {
         logical_time: EvalTime,
         session: &'a Session,
+        catalog_state: &'a CatalogState,
     },
     /// The expression is being prepared for evaluation in an AS OF or UP TO clause.
     AsOfUpTo,
+    /// The expression is being prepared for evaluation in a CHECK expression of a webhook source.
+    WebhookValidation {
+        /// Time at which this expression is being evaluated.
+        now: EpochMillis,
```

Comment:
```suggestion
        wall_time: DateTime<Utc>,
```

This is the type used by `PlanContext` and we should keep it the same.

Reply:
That's a great idea, updated!